src/SFML/Graphics/VertexBuffer.cpp

```diff
@@ -206,6 +206,7 @@ bool VertexBuffer::update(const Vertex* vertices, std::size_t vertexCount, unsig
 ////////////////////////////////////////////////////////////
 bool VertexBuffer::update(const VertexBuffer& vertexBuffer)
 {
+    (void) vertexBuffer;
```

Comment:
But we should at least be consistent. Either always use `(void)`, or `[[maybe_unused]]`, or comment. Might be worthwhile to see what the rest of the library already does.

Reply:
I chose to comment out those which are not used at all and cast to (void) those which are partially used.

`[[maybe_unused]]` as stated earlier is out of the capabilities.